book/src/tutorial-3.md

```diff
@@ -7,13 +7,18 @@ bindings to `bzip2` at compile time. The resulting bindings will be written to
 `$OUT_DIR/bindings.rs` where `$OUT_DIR` is chosen by `cargo` and is something
 like `./target/debug/build/bindgen-tutorial-bzip2-sys-afc7747d7eafd720/out/`.
 
+Note that the associated shared object to `bz2` is `libbz2.so`. In general, a `lib<name>.so` should be referenced in the build file by `<name>`.
```

Comment:
This is only true on UNIX, may be worth referencing that this would be bz2.dll on windows. Follow-up PR is fine of course.